resources/lang/vi/register.php

```diff
@@ -9,4 +9,10 @@
     'rememberme' => 'Ghi nhớ đăng nhập',
     'introduce' => 'Giới thiệu bản thân',
     'register' => 'Đăng kí',
+    'name-company' => 'Tên công ty',
```

Comment:
```suggestion
    'name_company' => 'Tên công ty',
```
sửa hết tên key bên dưới nhé